examples/WORKSPACE

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 Google Inc.  All rights reserved
```

Comment:
Where did these files come from? Did we modify them in any meaningful way?

If we didn't really modify them, couldn't we just point to the original location at which they are stored?